### ISSUE TITLE
Syntax highlighting when using ;

### ISSUE THREAD

**Issue by sujona** [krvajal/vscode-fortran-support#137]
For example 

```
if(this)then;      call bus(f1,p2,p3)
elseif(that)then;  call bus(e1,p2,p3)
else;              call bus(g1,p2,p3)
endif
```

gets incorrectly highlighted.

**Comment by pedro-ricardo** [comment]
Thank you, I'll look into it.
The problem is the `else` statement rule ... it forbids anything (except comments) to be in the same line.

Btw... I've never seen this particular usage of `if` and `;` in Fortran... but it's a valid syntax and should be correctly highlighted :+1:

**Comment by pedro-ricardo** [comment]
Fixed on #129

**Before**
![Screenshot_20190608_162137](https://user-images.githubusercontent.com/15893711/59151457-de465200-8a09-11e9-8efe-319d5dc17c69.png)
**After**
![Screenshot_20190608_162158](https://user-images.githubusercontent.com/15893711/59151460-e56d6000-8a09-11e9-9e43-c48384d739dd.png)

**Comment by sujona** [comment]
Nice! :+1:    
Yeah I use this kind of structure structure when suitable to get the alignment you see in the example.
